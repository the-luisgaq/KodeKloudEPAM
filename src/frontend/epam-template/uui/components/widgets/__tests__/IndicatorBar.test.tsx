import React from 'react';
import { IndicatorBar } from '../IndicatorBar';
import { render } from '@epam/uui-test-utils';

describe('IndicatorBar', () => {
    it('should be rendered correctly', () => {
        const { asFragment } = render(<IndicatorBar />);
        expect(asFragment()).toMatchSnapshot();
    });

    it('should be rendered correctly with props', () => {
        const { asFragment } = render(<IndicatorBar progress={ 20 } />);
        expect(asFragment()).toMatchSnapshot();
    });
});
