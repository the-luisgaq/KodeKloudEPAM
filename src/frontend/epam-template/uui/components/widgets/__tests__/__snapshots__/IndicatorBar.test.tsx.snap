// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndicatorBar should be rendered correctly 1`] = `
<DocumentFragment>
  <div
    class="uui-indeterminate_bar root root size-12"
  >
    <div
      class="progressBar"
    />
  </div>
</DocumentFragment>
`;

exports[`IndicatorBar should be rendered correctly with props 1`] = `
<DocumentFragment>
  <div
    class="uui-progress_bar root size-12 root container"
  >
    <div
      aria-valuemax="100"
      aria-valuemin="0"
      aria-valuenow="20"
      class="bar bar"
      role="progressbar"
      style="width: 20%;"
    />
  </div>
</DocumentFragment>
`;
